packages/editor-ui/src/views/SettingsSso.vue

```diff
@@ -78,6 +82,8 @@ const getSamlConfig = async () => {
 	ssoSettingsSaved.value = !!config?.metadata;
 };
 
+const testSucceeded = ref(false);
```

Comment:
what's the benefit of using a reactive reference over a regular old `let` here?

Reply:
My bad! Must've been tired EoD